pkgs/by-name/pi/pied/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter327,
+
+  gst_all_1,
+}:
+
+flutter327.buildFlutterApplication rec {
+  pname = "pied";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Elleo";
+    repo = "pied";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-I2p3GIb54r9r/phbKJsz/cFw1ECdwZ2RnCYVxjsHzg0=";
+  };
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  strictDeps = true;
+
+  buildInputs = [
+    gst_all_1.gst-plugins-base
+  ];
+
+  postInstall = ''
+    install -D flatpak/com.mikeasoft.pied.desktop -t $out/share/applications
+    install -D flatpak/com.mikeasoft.pied.png -t $out/share/pixmaps
+  '';
+
+  meta = {
+    description = "Pied makes it simple to install and manage text-to-speech Piper voices for use with Speech Dispatcher.";
```

Comment:
`meta.description` should not have an ending `.`
Also, it should ideally not have the package name itself in it.
See: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

The `.desktop` file has "Text-to-speech voice manager" as the comment.
Combining that together with the current description should give a shorter and more information dense description:

e.g.
"Piper text-to-speech voice manager for use with Speech Dispatcher"
or
"Piper text-to-speech voice manager for Speech Dispatcher"